CHANGES.rst

```diff
@@ -53,6 +53,8 @@ Specviz2d
 API Changes
 -----------
 
+- Removed some previously deprecated methods and arguments. [#2578]
```

Comment:
This needs to be more specific. User reading this change log entry needs to know exactly what was removed without having to read the diff here.

Reply:
There can even be multiple entries pointing to the same PR, as you see fit. For example, the wording can be something like this for one of them:

```rst
- Deprecated ``app.get_data_from_viewer`` is removed. Use ``viz_helper.get_data`` method instead. [#2578]
```

Reply:
Done - I left out the `show_continuum_marks` deprecation, since I think that was solely an internal thing, but I can add it for completeness if anyone thinks we need to.

Reply:
Yeah, no need to mention things users are not using directly. Thanks!